Keda.Cosmosdb.Scaler/src/Services/ExternalScalerService.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Threading.Tasks;
+using Google.Protobuf.Collections;
+using Grpc.Core;
+using Keda.Cosmosdb.Scaler.Protos;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.ChangeFeedProcessor;
+using Microsoft.Azure.Documents.Client;
+
+namespace Keda.Cosmosdb.Scaler.Services
+{
+    public class ExternalScalerService : ExternalScaler.ExternalScalerBase
+    {
+        private static ConsoleLogger logger = new ConsoleLogger();
+        public override async Task<IsActiveResponse> IsActive(ScaledObjectRef request, ServerCallContext context)
+        {
+            bool isActive = false;
+            try
+            {
+                var trigger = CreateTriggerFromMetadata(request.ScalerMetadata);
+                var workToBeDone = await GetEstimatedWork(trigger);
+
+                isActive = workToBeDone > 0;
+
+                if (isActive)
+                {
+                    logger.Log(string.Format("Activating to 1 for cosmosDB account {0} database {1} collection {2}", trigger.AccountName, trigger.DatabaseName, trigger.CollectionName));

```

Comment:
Using `ILogger` will allow you to pass params directly to log message. Also, use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) instead of `string.Format`

Reply:
We should actually use structured logging instead of format or interpolation